README.rst

```diff
@@ -1,334 +1,503 @@
-================
+
 pytest-datafiles
-================
+****************
 
-.. image:: https://img.shields.io/travis/omarkohl/pytest-datafiles.svg
-        :target: https://travis-ci.org/omarkohl/pytest-datafiles
+https://travis-ci.org/omarkohl/pytest-datafileshttps://coveralls.io/github/omarkohl/pytest-datafiles?branch=masterhttps://pypi.python.org/pypi/pytest-datafileshttps://codeclimate.com/github/omarkohl/pytest-datafiles
```

Comment:
that is (unfortunately) the result of generating the rst  from a rst ...

Reply:
I would also be fine with having an "examples" directory and just linking to the examples in the README instead of this more complex setup generating the README etc.

Reply:
done